app/code/Magento/Bundle/Test/Mftf/Test/StorefrontCustomerSelectAndSetBundleOptionsTest.xml

```diff
@@ -22,6 +22,9 @@
             <actionGroup ref="LoginAsAdmin" stepKey="login"/>
             <createData entity="SimpleProduct2" stepKey="simpleProduct1"/>
             <createData entity="SimpleProduct2" stepKey="simpleProduct2"/>
+
+            <wait stepKey="waitBeforeIndexer" time="60"/>
```

Comment:
Why we need to add wait for 1 minute in a lot of tests? Tests run will just became longer, can’t we prevent it?

Reply:
It's temporary solution that will disappear when we will have <magentoCron command available (Monday). 

It is because of running different tests that run cron jobs at the same time. 

Can draw a graph explaining the issue.

Reply:
So I believe this is issue of mftf, Is there any link to it?

Reply:
Assume that when you run 4 workers of tests, and these tests are ran simultaneously - all 4 are running <createData and cron:run --group=index in 30 seconds. As a result - the first one gets the data indexer, but all the rest - not, as cron is waiting for 60 seconds before doing another round of indexing. That's the reason why it works randomly - it's just related with overlapping actions.

Reply:
I removed once again waiting for Indexers, maybe - by coincidence - this will work somehow.

Reply:
@ihor-sviziev - The new feature for running Cron Jobs is mentioned in MFTF Changelog:
https://github.com/magento/magento2-functional-testing-framework/blob/master/CHANGELOG.md

https://github.com/magento/magento2-functional-testing-framework/pull/538 -- FEATURE: command to execute Cron Jobs